client/src/components/Collections/common/CollectionEditView.vue

```diff
@@ -1,19 +1,152 @@
+<script setup lang="ts">
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faBars, faCog, faDatabase, faTable, faUser } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import axios from "axios";
+import { BAlert, BTab, BTabs } from "bootstrap-vue";
+import { storeToRefs } from "pinia";
+import { computed, ref } from "vue";
+
+import { DatatypesProvider, DbKeyProvider, SuitableConvertersProvider } from "@/components/providers";
+import { useConfig } from "@/composables/config";
+import { useCollectionAttributesStore } from "@/stores/collectionAttributesStore";
+import { useHistoryStore } from "@/stores/historyStore";
+import localize from "@/utils/localization";
+import { prependPath } from "@/utils/redirect";
+import { errorMessageAsString } from "@/utils/simple-error";
+
+import { HistoryContentBulkOperationPayload, updateHistories } from "./services";
+
+import ChangeDatatypeTab from "@/components/Collections/common/ChangeDatatypeTab.vue";
+import DatabaseEditTab from "@/components/Collections/common/DatabaseEditTab.vue";
+import SuitableConvertersTab from "@/components/Collections/common/SuitableConvertersTab.vue";
+import LoadingSpan from "@/components/LoadingSpan.vue";
+
+library.add(faBars, faCog, faDatabase, faTable, faUser);
+
+interface Props {
+    collectionId: string;
+}
+
+const props = defineProps<Props>();
+
+const { config, isConfigLoaded } = useConfig(true);
+const collectionAttributesStore = useCollectionAttributesStore();
+
+const historyStore = useHistoryStore();
+const { currentHistoryId } = storeToRefs(historyStore);
+
+const jobError = ref(null);
+const errorMessage = ref("");
+const noQuotaIncreaseMessage = ref("Your quota will not increase.");
+const newCollectionMessage = ref("This will create a new collection in your History.");
+const infoMessage = ref("This will create a new collection in your History. Your quota will not increase.");
+const expectWaitTimeMessage = ref("This operation might take a short while, depending on the size of your collection.");
+
+const attributesData = computed(() => {
+    return collectionAttributesStore.getAttributes(props.collectionId);
+});
+const databaseKeyFromElements = computed(() => {
+    return attributesData.value?.dbkey;
+});
+const datatypeFromElements = computed(() => {
+    return attributesData.value?.extension;
+});
+
+function updateInfoMessage(strMessage: string) {
+    infoMessage.value = strMessage;
+}
+
+// TODO: Replace with actual datatype type
+async function clickedSave(attribute: string, newValue: any) {
+    const url = prependPath(`/api/dataset_collections/${props.collectionId}/copy`);
```

Comment:
This API endpoint is available in the client schema, so it would be cool to add the corresponding fetcher to `client/src/api/datasetCollections.ts` and use it here.